packages/google_maps_flutter/google_maps_flutter_web/lib/src/convert.dart

```diff
@@ -131,9 +131,35 @@ gmaps.MapOptions _configurationAndStyleToGmapsOptions(
 
   options.mapId = configuration.mapId;
 
+  // Convert the color scheme, if any, and set it
+  final gmaps.ColorScheme? jsColorScheme = _gmapTypeColorSchemeForPluginColor(
+    configuration.colorScheme,
+  );
+  if (jsColorScheme != null) {
+    options.colorScheme = jsColorScheme;
+  }
+
   return options;
 }
 
+/// Converts a [MapColorScheme] into a [gmaps.ColorScheme]
+gmaps.ColorScheme? _gmapTypeColorSchemeForPluginColor(MapColorScheme? scheme) {
+  // Guard clause for null
+  if (scheme == null) {
+    return null;
+  }
+
+  // Map to gmaps.ColorScheme
+  switch (scheme) {
+    case MapColorScheme.dark:
+      return gmaps.ColorScheme.DARK;
+    case MapColorScheme.light:
+      return gmaps.ColorScheme.LIGHT;
+    case MapColorScheme.followSystem:
+      return gmaps.ColorScheme.FOLLOW_SYSTEM;
+  }
```

Comment:
This needs fallthrough handling; see the method just below it for an example and an explanation of why.